### DISCUSSION TITLE
can't write certain options

### DISCUSSION THREAD

**Discussion by bigmac5753** [cedricp/ddt4all#1156] [Q&A]
how can I write to these options?

(I know it's in demo mode, I get the same in connected mode)

![unnamed](https://github.com/cedricp/ddt4all/assets/17450308/00b66323-7b15-4187-a100-7b999b82cd40)

**Comment by AndyIreland** [comment]
hi, its because that ecu parameter can only be read, not modified - there are no boxes on the 'write' for you to put your parameters into. 
Whether you can use the manual command 

![image](https://github.com/cedricp/ddt4all/assets/49172217/9e64d436-96c2-410c-aa7e-732688e7ff55)

is another matter, you might be able to, but its a risk.

**Comment by bigmac5753** [comment]
I found the options to read, but not write, I guess it can't be done.

![Screenshot (2)](https://github.com/cedricp/ddt4all/assets/17450308/e27c190d-26b9-42da-b27d-7f7c8170765c)
![Screenshot (3)](https://github.com/cedricp/ddt4all/assets/17450308/db4b6af1-f83b-4d41-9dba-2002e92eb42a)

**Comment by AndyIreland** [comment]
I think thats the case - especially if you have tried the manual button too and there is no 'write' data for those settings
